docs/tutorial-code/redis.go

```diff
@@ -78,11 +78,40 @@ func populateMetrics(ms *metric.MetricSet) error {
 	if err != nil {
 		return err
 	}
-
 	ms.SetMetric("net.connectionsReceivedPerSecond", metricValue, metric.RATE)
+
+	cmd = exec.Command("/bin/sh", "-c", "redis-cli info | grep uptime_in_seconds:")
```

Comment:
Now I'm thinking that I overcomplicated the code. Instead of creating a new metrics I could use an existing metric `net.connectionsReceivedPerSecond` and create an event like this:
```
err = integration.AddEvent("More than x connection received per second", "notification")
```

Reply:
Code updated in the second commit.